rasa/cli/data.py

```diff
@@ -99,4 +103,5 @@ def validate_files(args):
     )
 
     validator = loop.run_until_complete(Validator.from_importer(file_importer))
-    validator.verify_all()
+    everything_is_alright = validator.verify_all(not args.fail_on_warnings)
+    sys.exit(not everything_is_alright)  # fail with non zero if there are issues
```

Comment:
the code might speak for itself better and not require a comment if we
`sys.exit(0) if everything_is_alright else sys.exit(1)`

Reply:
great suggestion :rocket:

```suggestion
    sys.exit(0) if everything_is_alright else sys.exit(1)
```